app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java

```diff
@@ -1216,6 +1218,7 @@ private void logInTestUser() {
         _solo.enterText(_solo.getEditText("Password"), _testUser.getPassword());
 
         _solo.clickOnButton(LOGIN_TEXT);
-        _solo.sleep(4000);
+        // Wait for Profile fragment to load
```

Comment:
This was changed to check for the profile fragment rather than just wait an arbitrary time.